ui/modal/text_input_modal.go

```diff
@@ -87,9 +90,14 @@ func (tm *TextInputModal) SetCancelable(min bool) *TextInputModal {
 	return tm
 }
 
+func (tm *TextInputModal) SetTextWithTemplate() *TextInputModal {
+	tm.textCustomTemplate = allowUnspendUnmixedAcct(tm.Theme)
+	return tm
+}
```

Comment:
This section here needs to be dynamic just in case we need to use another template.

```suggestion
func (tm *TextInputModal) SetTextWithTemplate(template string) *TextInputModal {
	switch template {
	case AllowUnmixedSpendingTemplate:
		tm.textCustomTemplate = allowUnspendUnmixedAcct(tm.Theme)
	}

	return tm
}
}
```